### DISCUSSION TITLE
[Support]: Property auth is not allowed

### DISCUSSION THREAD

**Discussion by masalski** [blakeblackshear/frigate#14441] [General Support] [answered]
### Describe the problem you are having

Hello,
I am setting up Frigate as a docker container in Debian.
I was trying to setup authentication, but it gives me error when I try to configure it:
"Property auth is not allowed"
It looks like this is a new feature of version 0.14.1 release. 
Unfortunately, I cannot use "latest" for docker...
Are these correct assumptions and is there a plan 0.14.1 to be available for docker as well soon?
Thank you!

### Version

stable 0.13.2

### Frigate config file

```yaml
n/a
```

### Relevant Frigate log output

```shell
n/a
```

### Relevant go2rtc log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Install method

Docker CLI

### docker-compose file or Docker CLI command

```yaml
n/a
```

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

n/a

**Comment by NickM-27** [comment]
Auth is only available in 0.14

**Comment by hawkeye217** [comment, marked as answer]
0.14 is already available via docker. https://github.com/blakeblackshear/frigate/releases

**Comment by masalski** [comment]
Thank you both for the replies! I was able to run release 0.14.1 in docker successfully. Let me explain why I ended initiated this discussion... I was following the installation documentation and for docker it is pointed to use "stable" release (which it looks it is 0.13.2). I was dumb enough to try running "latest" which throws some manifest error. Not sure why I decided that not all releases are docker compatible. Then decided to ask you... Thanks again!
